analyticConfigurations/computed_metric-elasticache.json

```diff
@@ -9,7 +9,7 @@
       "elementName" : null,
       "elementType" : null,
       "elementTags" : null,
-      "metricMatches" : null
+      "metricMatches" : "aws.elasticache.freeablememory"
```

Comment:
@jemassey does this mean that only elements with the aws.elasticache.freeablememory metric will have these computed metric applied to them? @TheConnMan do all elasticache metrics have that metric?

Reply:
Yes, all ElastiCache nodes and clusters have this metric.